enhancements/oc/day2-add-nodes.md

```diff
@@ -0,0 +1,262 @@
+---
+title: simplify-and-unify-day2-add-node
+authors:
+  - "@andfasano"
+  - "@zaneb"
+approvers: 
```

Comment:
From the [template](https://github.com/openshift/enhancements/blob/master/guidelines/enhancement_template.md?plain=1#L7):

> # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.